modules/accessing-s3-compatible-object-storage-with-self-signed-certificates.adoc

```diff
@@ -0,0 +1,80 @@
+endif::[]
+
+ifdef::disconnected[]
+Alternatively, you can add a custom CA bundle by using the OpenShift console, as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/working-with-certificates_certs#adding-a-ca-bundle_certs[Adding a CA bundle].
+endif::[]
+
+endif::[]
+
+ifdef::cloud-service[]
+Alternatively, you can add a custom CA bundle by using the OpenShift console, as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs#adding-a-ca-bundle_certs[Adding a CA bundle].
+endif::[]
+
+endif::[]
+
+.Prerequisites
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+* You have downloaded and installed the OpenShift command-line interface (CLI). See link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/cli_tools/openshift-cli-oc#installing-openshift-cli[Installing the OpenShift CLI^].
+* You have an object storage solution or database deployed in your OpenShift cluster.
+
+.Procedure
+. In a terminal window, log in to the OpenShift CLI as shown in the following example:
++
+[source]
+----
+oc login api.<cluster_name>.<cluster_domain>:6443 --web
+----
+. Run the following command to fetch the current {productname-short} trusted CA configuration and store it in a new file.
++
+[source]
+----
+oc get dscinitializations.dscinitialization.opendatahub.io default-dsci -o json | jq -r '.spec.trustedCABundle.customCABundle' > /tmp/my-custom-ca-bundles.crt
+----
+. Add the cluster's `kube-root-ca.crt` to the {productname-short} trusted CA configuration:
++
+[source]
+----
+oc get configmap kube-root-ca.crt -o jsonpath="{['data']['ca\.crt']}" >> /tmp/my-custom-ca-bundles.crt
+----
+. Update the {productname-short} trusted CA configuration to trust certificates issued by the certificate authorities in `kube-root-ca.crt`:
++
+[source]
+----
+oc patch dscinitialization default-dsci --type='json' -p='[{"op":"replace","path":"/spec/trustedCABundle/customCABundle","value":"'"$(awk '{printf "%s\\n", $0}' /tmp/my-custom-ca-bundles.crt)"'"}]'
+----
+
+.Verification
+* You can successfully deploy components that are configured to use object storage solutions or databases that are deployed in the OpenShift cluster. For example, a pipeline server that is configured to use a database deployed in the cluster starts successfully.
+
+ifndef::upstream[]
+[NOTE]
+====
+You can test your new certificate configuration by following the steps in the {productname-short} tutorial fraud detection example to run a script to install local object storage buckets and create connections, and then enable data science pipelines.
```

Comment:
The sentence on line 72 could be revised to:

You can verify your new certificate configuration by running the script provided in the {productname-short} fraud detection tutorial. This script installs local object storage buckets, creates connections, and enables data science pipelines.

Reply:
Updated.